modules/015-admission-policy-engine/crds/doc-ru-security-policy.yaml

```
@@ -6,53 +6,53 @@ spec:
           description: |
             Описывает политику безопасности для кластера.
 
-            Каждый ресурс `SecurityPolicy` описывает правила для объектов в кластере.
+            Каждый ресурс `SecurityPolicy` задает правила для объектов в кластере.
           properties:
             spec:
               properties:
                 enforcementAction:
                   description: |
-                    Действие, которое будет выполнено по результатам проверки ограничений:
+                    Действие, которое будет выполнено из-за проверки ограничений:
```

```suggestion
                    Действие, которое будет выполнено из-за проверки ограничений:
```
Спорные моменты: 
- по сути же идет проверка ограничения согласно установленным правилам (политикам), потом, пос результаатм этой проверки выполняется что-то. Если удовлетворояет, то выполнено ничего не будет. Т.е. "из-за" тут  мне кажется неуместно.
- Еще момент.  У нас в заголовке списка указано сто это список действий, но при DryRun и Warn пишем что действие отсутствует... Не клеится.

---

@MariaMakeeva ^^^

---

```suggestion
                    Действие, которое будет выполнено после проверки ограничений:
```

---

По поводу списка действий. может быть подзаголовок изменить на "список действий, при необходимости", например.

---

Действие (при необходимости), которое будет выполнено после поверки ограничений

---

> 

Дак я же написал выше, что если удовлетворяет политики, то никакого действия выполнено не будет.  Т.е. не факт что оно будет выполнено ни "после", ни "по результату". Но по результату про идее более правильно, т.к. смысл такой, что будет какой-то результат, и в зависимости от него - будет действие. А если "после" - то подразумевается что будет безусловно.

---

Да, Артём, спасибо. Ты подал хорошую мысль, что там не просто список действий, а что действия будут выборочными.